internal/commands/database/session/session.go

```diff
@@ -0,0 +1,16 @@
+package databasesession
+
+import (
+	"github.com/UpCloudLtd/upcloud-cli/v2/internal/commands"
+)
+
+// BaseSessionCommand creates the base "session" command
+func BaseSessionCommand() commands.Command {
```

Comment:
If the interface is equal to removed `connection` commands, we could use that as an alias here to make this non-breaking change. Not sure if that would cause confusion, though.

Reply:
I think it's better to keep them separate in this case. Good idea to consider in any case.